OSDOCS#12433: Add disconnected support for OLM v1 (#86791)
openshift/openshift-docs: modules/oc-mirror-updating-cluster-manifests.adoc

// Module included in the following assemblies:
//
// * disconnected/mirroring/installing-mirroring-disconnected.adoc

:_mod-docs-content-type: PROCEDURE
[id="oc-mirror-updating-cluster-manifests_{context}"]
= Configuring your cluster to use the resources generated by oc-mirror

After you have mirrored your image set to the mirror registry, you must apply the generated `ImageContentSourcePolicy`, `CatalogSource`, and release image signature resources into the cluster.

The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry. The `CatalogSource` resource is used by {olmv0-first} to retrieve information about the available Operators in the mirror registry. The release image signatures are used to verify the mirrored release images.

[NOTE]
====
{olmv1} uses the `ClusterCatalog` resource to retrieve information about the available cluster extensions in the mirror registry.

> The oc-mirror plugin v1 does not generate `ClusterCatalog` resources automatically, and they must be manually created. For more information on creating and applying `ClusterCatalog` resources, see "Adding a catalog to a cluster" in "Extensions".

Reviewer: Passive/active?
```suggestion
The oc-mirror plugin v1 does not generate `ClusterCatalog` resources automatically, and you must manually created. For more information on creating and applying `ClusterCatalog` resources, see "Adding a catalog to a cluster" in "Extensions".
```
